pkg/terraform/stages/ovirt/stages.go

```diff
@@ -56,6 +58,10 @@ func extractOutputHostAddresses(s stages.SplitStage, directory string, ic *types
 			return
 		}
 		bootstrapIP = ip
+		logrus.Debug("Found bootstrapIP: %s", bootstrapIP)
+	} else {
+		returnErr = errors.New("bootstrap vm id not found in Terraform outputs")
+		return
```

Comment:
You don't want the return here. Only the output file from the bootstrap stage will have `bootstrap_vm_id`. The output file from the cluster stage will have `control_plane_vm_ids`.